api/camunda/camunda-openapi.yaml

```
@@ -966,7 +972,9 @@ paths:
         - Tenant
       operationId: assignRoleToTenant
       summary: Assign a role to a tenant
-      description: Assign a single role to a specified tenant.
+      description: |
+        Assigns a role to a specified tenant.
+        Users, Clients or Groups, that have the role assigned, will get access to the tenant's data and can perform actions according to their authorizations.
```

```suggestion
        Users, clients, or groups that have the role assigned, will get access to the tenant's data and can perform actions according to their authorizations.
```

I would suggest this edit but not blocking to this PR.